css-ui-3/caret-color-017.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: caret-color transition</title>
+<link rel="author" title="Manuel Rego Casasnovas" href="mailto:rego@igalia.com">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#caret-color">
+<link rel="help" href="https://www.w3.org/TR/css3-transitions/#transition">
+<meta name="flags" content="interact animated">
```

Comment:
I don't remember whether this is documented anywhere, but I think we've recently decided that it was not necessary to distinguish between `interact` and `animated` anymore, and that `interact` alone was good enough to represent both.

@gsnedders , can you confirm?